documentation/docs/03-loading.md

```diff
@@ -78,7 +78,7 @@ SvelteKit's `load` receives an implemention of `fetch`, which has the following
 
 > Code called inside `load` blocks:
 >
-> - should use the SvelteKit-provided [`fetch`](#loading-input-fetch) wrapper rather than using the native `fetch`
+> - should use the SvelteKit-provided [`fetch`](#loading-input-fetch) wrapper rather than using the native fetch as the latter will make the request twice each during SSR and hydration
```

Comment:
I like that this explains why, but it could be said a bit more clearly as it somewhat sounds like twice during SSR and twice during hydration. also not sure why you'd take fetch out of backticks. 
```suggestion
> - should use the SvelteKit-provided [`fetch`](#loading-input-fetch) wrapper rather than using the native `fetch` because SvelteKit's version has built-in hydration support to avoid making a second request on the client.
```